native/src/snappyclient/cpp/impl/ControlConnection.cpp

```diff
@@ -123,7 +123,7 @@ const boost::optional<ControlConnection&> ControlConnection::getOrCreateControlC
     }
   }
 
-  if (allConnSize == 0) { // first attempt of creating connection
+    // first attempt of creating connection
```

Comment:
Since you haven't changed the indentation after the removal of if() block, so its better to add an empty opening and closing brace:
```
{
  // first attempt of creating connection
  ...
}
```

Reply:
done